test/vinyl-luatest/box_cfg.lua

```diff
@@ -0,0 +1,14 @@
+return {
+    work_dir = os.environ()['VARDIR'],
+    memtx_memory = 512 * 1024 * 1024,
+    memtx_max_tuple_size = 4 * 1024 * 1024,
```

Comment:
Let's drop memtx parameters from the config - the test should work just fine with the defaults. After all, we test vinyl here, not memtx.